client/src/app/app-routing.module.ts

```diff
@@ -2,10 +2,12 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 import { PaiComponent } from './components';
+import { ContactUsComponent } from './components/contact-us/contact-us.component';
```

Comment:
You can remove this line, and import the `ContactUsComponent` component in line 4, with `PaiComponent` (because you add the `ContactUsComponent` import to `app/components/index.ts` file).